include/circt/Dialect/FIRRTL/Passes.td

```diff
@@ -90,6 +90,14 @@ def BlackBoxMemory : Pass<"firrtl-blackbox-memory", "firrtl::CircuitOp"> {
   ];
 }
 
+def EmitMetadata : Pass<"firrtl-emit-metadata", "firrtl::CircuitOp"> {
```

Comment:
Is this a general to all Chisel users pass or sifive specific?  If it is sifive specific, you might name this "firrtl-emit-sifive-metadata" or something like that.